runtime.md

```diff
@@ -14,4 +14,45 @@ Runs a process in a container. Can be invoked several times.
 
 Not sure we need that from runc cli. Process is killed from the outside.
 
-This event needs to be captured by runc to run onstop event handlers.
+This event needs to be captured by oc to run onstop event handlers.
+
+## Hooks
+Hooks allow one to run code before/after various lifecycle events of the container.
+The state of the container is passed to the hooks over stdin, so the hooks could get the information they need to do their work.
+
+Hook paths are absolute and are executed from the host's filesystem.
+
+### Pre-start
+The pre-start hooks are called after the container process is started but before the user supplied command is executed.
```

Comment:
I would change this to `after the container's namespaces are created` and not say started because it's confusing with the hooks name as prestart